CHANGES_NEXT_RELEASE

```diff
@@ -33,4 +33,5 @@ Add:  Hardening: mongoBackend checking for field existence and proper type to av
 Add:  githash in --version (Issue #1363)
 Add:  Added CLI argument for notification mode. Default mode is 'transient' (previous version used 'permanent' implicitly) (Issue #1370)
 Add:  Added CLI argument for connection memory limit.  Default value is 64Kb (Issue #1384)
+Fix:  Separate semahpore for subscription cache, so that sub cache is independent of mutex policy (Issue #1401)
```

Comment:
As it is a "separate issue", I thought it convenient to mark it here.
What we perhaps should do is to make sure the issue 1401 is clearly marked as "related" to 1308